main.py

```diff
@@ -3,82 +3,38 @@
 import tkinter as tk
 from tkinter import simpledialog
 from utils.node_class import Node
+from utils.player_class import Player  # Import the Player class
+from utils.path_class import Path
 from password_cracker.user_interface import PasswordCracker
+PROBLEMS = [PasswordCracker()]
 
+# Fonts
+font = pygame.font.Font(None, 36)
 
-PROBLEMS = [PasswordCracker()]
+# Define game states
+MENU = 0
+GAME = 1
+game_state = MENU
 
-#background = random.choice([pygame.image.load("sl_031420_28950_10.jpg"), pygame.image.load("abstract-techno-background-with-connecting-dots-circuit-board-image.jpg")])
+#background = random.choice([pygame.image.load("Menu_BG.jpg"), pygame.image.load("abstract-techno-background-with-connecting-dots-circuit-board-image.jpg")])
```

Comment:
can you remove this comment. I think it is unnecessary since I have added the background on my other PR.